homeassistant/components/knx/validation.py

```diff
@@ -40,11 +40,11 @@ def ga_validator(value: Any) -> str | int:
     if isinstance(value, (str, int)):
```

Comment:
Side note: This should preferably be inverted and raise if true so we can outdent below.